src/applications/Machine.java

```diff
@@ -1,6 +1,7 @@
 package applications;
 
 import dataStructures.LinkedQueue;
+import jdk.jfr.Event;
```

Comment:
I bet this is an autocomplete thing, and that you almost certainly don't want it.